src/mq.c

```diff
@@ -436,34 +436,20 @@ int mq_set_queue_to_consume(amqp_bytes_t queue,
 		return -1;
 	}
 
-	/* Start a queue consumer */
```

Comment:
I think we can abstract it. Since the read_cb will just be called if the queue is set as a consumer.
But I will change the function `mq_set_queue_to_consume` to `mq_bind_queue`